linode_api4/objects/account.py

```diff
@@ -39,8 +39,121 @@ class Account(Base):
         "tax_id": Property(mutable=True),
         "capabilities": Property(),
         "credit_card": Property(),
+        "active_promotions": Property(),
+        "active_since": Property(),
+        "balance_uninvoiced": Property(),
+        "billing_source": Property(),
+        "euuid": Property(),
     }
 
+    def event_seen(self, event_id):
+        """
+        Marks all Events up to and including this Event by ID as seen.
+        """
+
+        self._client.post(
+            "{}/events/{}/seen".format(self.api_endpoint, event_id), model=self
+        )
```

Comment:
Can this be handled by updating the `Event` object directly?

Reply:
It seems that it is already in [`linode_client.py`](https://github.com/linode/linode_api4-python/blob/53ddf8559e449c80c5b00cb51d9718980051a141/linode_api4/linode_client.py#L665)

Reply:
Good catch! I didn't realize that this is already handled. I will remove the duplicate implementation.